yew/src/virtual_dom/vlist.rs

```diff
@@ -172,6 +439,950 @@ mod tests {
             </div>
         };
     }
+
+    fn new_keyed_vtag<T: Into<Cow<'static, str>>, K: ToString>(tag: T, key: K) -> VTag {
+        let mut vtag = VTag::new(tag.into());
+        vtag.key = Some(Key::from(key.to_string()));
+        vtag
+    }
+
+    #[cfg(feature = "web_sys")]
+    fn inner_html(element: &Element) -> String {
+        element.inner_html()
+    }
+
+    #[cfg(feature = "std_web")]
+    fn inner_html(element: &Element) -> String {
+        use stdweb::unstable::TryInto;
+        use stdweb::web::{HtmlElement, IHtmlElement};
+        let html_element: HtmlElement = element
+            .clone()
+            .try_into()
+            .expect("Failed to convert element into HtmlElement");
+        html_element.inner_text()
+    }
+
+    #[test]
+    fn vlist_vdiff_apply_non_keyed_from_none_works_with_all_vnode_types_as_children() {
```

Comment:
Could you please remove the non_keyed tests? I think those are covered from https://github.com/yewstack/yew/pull/1279

Reply:
Removed.